arci-ros/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
`arci-ros` includes the file which is from BSD lisences in arci-ros/ros_msgs dir.
It contains LICENSE.BSD, which I created.

Do you think that is it safe to say that this is Apache-2?
Shouldn't we move LICENSE.BSD to `arci-ros/`?

Reply:
Hmm, it seems LICENSE.BSD is BSD 2-clause, so I think it's okay to distribute it as part of the Apache-2 licensed software. (Disclaimer: I am not a lawyer) 
However, I think we need to add a file like rust-lang/rust's [COPYRIGHT](https://github.com/rust-lang/rust/blob/78e22069d018e83915201c8a218a0a94227f6420/COPYRIGHT) to the arci-ros crate regarding the inclusion of source files with different licenses.

refs: 
- https://www.apache.org/legal/resolved.html (BSD 2-clause is in category A)
- https://infra.apache.org/licensing-howto.html (see "Bundling permissively-licensed dependencies" section)

Reply:
I see. Anyway this PR improves current situation, let's merge this.